### ISSUE TITLE
Documentation (Theory manual) related to damping

### ISSUE THREAD

**Issue by samyogshr** [idaholab/mastodon#327]
Simulation of a constant damping ratio: In this heading, equation for zeta has (f1 f1) term, should that be (f1 f2)?

In equation (3) of theory manual (and the subsequent equation), should the right hand side of the equation have (2 pi f) term instead of (f)?

**Comment by somu15** [comment]
Yes, your first comment seems to be right. That should be (f1 f2).

Equation (3) is defined in terms of cyclic frequencies. So, having f is correct.

@samyogshr Would you like to submit a PR changing (f1 f1) to (f1 f2)? If not, let me know and I can create one.

**Comment by samyogshr** [comment]
@somu15 Please take another look at equation (3), the next equation (i.e. I = ...) and how it is used to derive zeta and eta values, which have 2 pi terms.

Did you mean f is in rad/s in equation (3)?
